### ISSUE TITLE
Find and fix non-translatable strings

### ISSUE THREAD

**Issue by scribblemaniac** [pencil2d/pencil#1275]
One of our translators has reported that some of the strings in the user interface remain untranslatable. These include the following unconfirmed strings:
- "Menu item text". Don't bother fixing this as the bug that caused it to be shown is fixed I believe, and the undo/redo rewrite will supersede it anyway.
- "Play" in Animation menu
- "Autosave documents" in Files preferences
- "Short scrub" in Timeline preferences
- Everything in the Flip and Roll section of the Timeline preferences
- Shortcuts preferences (which are also not exactly in plain English either). It's probably worth discussing if these should be translated, but at the moment it seems to me like it would be a good idea.

Additionally there may be other untranslated strings around, particularly with new features. One idea I have to make finding these easier is to have a special language that can be enabled at compile time (so that it is not in the releases). This language would just have say 'X's in place of all the translatable text. This would make it very easy to see what text, and what parts of text are being translated and which ones aren't. Ideally this should be generated from the `translations/pencil.ts`, replacing all the letters with the substitute character and leaving the %1,  %2, etc. arguments in place.

**Comment by davidlamhauge** [comment]
The text in the Shortcuts preferences are made by removing the first three characters from the shortcuts name: 'CmdNewFile' becomes 'NewFile' and so on. All shortcut names start with 'Cmd'.

The rest of the strings you mention, have been translated to Danish without problems, so it puzzles me that they can be untranslatable?

**Comment by scribblemaniac** [comment]
> The text in the Shortcuts preferences are made by removing the first three characters from the shortcuts name: 'CmdNewFile' becomes 'NewFile' and so on. All shortcut names start with 'Cmd'.

Yes I was generally aware of the technical details, but just because that's the way we do it now, does not mean that it should be done that way. I can imagine that it would be difficult for anyone who does not speak English to set their shortcuts as it is right now.

> The rest of the strings you mention, have been translated to Danish without problems, so it puzzles me that they can be untranslatable?

Like I said, they are unconfirmed, so it's possible the translator was mistaken and they are just not translated in that particular language.

**Comment by J5lx** [comment]
I just checked again and none of the issues from the original post are still present in current master. I also ran the program with the pseudo translations generated by Transifex and did not spot any untranslated strings still left, so I reckon this issue can be closed.
